### PR TITLE
dedupe

### PR DESCRIPTION
**Pull request by bwalsh**
This PR addresses duplicate vertexes and edges found in g2p, compounds and phenotypes:
* amends unit test to ensure all edges and vertexes are uniq
* adds integration test to check processed files for dups

### CODE REVIEW

File: src/bmeg/maf/maf_transform.py [hunk 1]
```diff
@@ -197,18 +195,19 @@ def maf_convert(self, emitter, mafpath, source,
...
-                                          Aliquot.make_gid(callset.tumor_aliquot_id)
+                                          Aliquot.make_gid(callset.tumor_aliquot_id),
                                           )
 
                 # create edge to gene
                 gene_gid = self.create_gene_gid(line)
                 if gene_gid:
                     emitter.emit_edge(AlleleIn(), allele.gid(), gene_gid)
             except Exception as exc:
+                logging.exception(exc)
```
Comment (adamstruck) [inline src/bmeg/maf/maf_transform.py]: are both of these log lines needed?

File: src/bmeg/vertex.py [hunk 2]
```diff
@@ -28,21 +28,17 @@ def from_dict(cls, data):
 class Callset(Vertex):
     tumor_aliquot_id: str
     normal_aliquot_id: Union[None, str]
-    call_method: str
```
Comment (adamstruck) [inline src/bmeg/vertex.py]: why is this being removed? I think we need a way to distinguish call sets on the same set of aliquots that were analyzed using different methods.

File: transform/ccle/samples.py [hunk 3]
```diff
@@ -5,6 +5,8 @@
 from bmeg.enrichers.phenotype_enricher import phenotype_factory
 from pydash import is_blank
 
+SKIP_RENAME = ['TT_OESOPHAGUS', 'TT_THYROID']
```
Comment (adamstruck) [inline transform/ccle/samples.py]: Why are these cases being skipped?

File: src/bmeg/vertex.py (hunk 2 above)
Comment (adamstruck) [reply src/bmeg/vertex.py]: Talked to Kyle, sounds like we will keep `source` on `Callset` and put `call_methods` as an array of strings on the `AlleleCall` edges.

File: src/bmeg/maf/maf_transform.py (hunk 1 above)
Comment (bwalsh) [reply src/bmeg/maf/maf_transform.py]: done

**Comment by bwalsh** [comment]
see #212 
https://github.com/bmeg/bmeg-etl/issues/212#issuecomment-428822411

**Pull request merged**